vernac/comInductive.ml

```diff
@@ -283,15 +283,16 @@ let interp_mutual_inductive (paramsl,indl) notations cum poly prv finite =
   let sigma, arities = List.fold_left_map (fun sigma -> interp_ind_arity env_params sigma) sigma indl in
 
   let fullarities = List.map (fun (c, _, _) -> Term.it_mkProd_or_LetIn c ctx_params) arities in
+  let fullearities = List.map EConstr.of_constr fullarities in
```

Comment:
If we remove the `EConstr.Unsafe.to_constr` in `interp_ind_arity` (and use econstr `it_mkProd_or_LetIn` and modify `push_types` to be econstr) this should disappear.

Reply:
Done, but futher cleanup with arities seems to be needed.